jenkins/raspberrypi/run_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+chmod -R 777 $PWD/drop
```

Comment:
I'm pretty sure you need a set -e to return errors out of this file to validate failures.